src/applications/disability-calculator/components/DisabilityRatingCalculator.jsx

```diff
@@ -136,8 +143,8 @@ export default class DisabilityRatingCalculator extends React.Component {
               indx={idx}
               ratingRef={this.ratingRef}
               handleRemoveRating={this.handleRemoveRating}
-              canDelete={idx > 1}
               ref={this.setRef}
+              canDelete={this.state.ratings.length}
```

Comment:
If you follow my comment below about using `idx`, we don't need this `canDelete` prop.